test/Microsoft.TestPlatform.AcceptanceTests/DotnetArchitectureSwitchTests.cs

```diff
@@ -36,7 +37,7 @@ public static void ClassCleanup()
         {
             try
             {
-                Directory.Delete(privateX64Installation, true);
+                new DirectoryInfo(privateX64Installation).Parent.Delete(true);
```

Comment:
Why Parent?

Reply:
In the init we do a temp dir with a subfolder named `x64` and we keep a reference to the sub-folder, so I want to clean the right folder (i.e. the parent).